pkg/generator/caches.go

```diff
@@ -47,13 +47,23 @@ type Caches struct {
 }
 
 func NewCaches(ctx context.Context, logger *zap.SugaredLogger, kubernetesClient kubeclient.Interface, extAuthz bool) (*Caches, error) {
+	statusVirtualHost := statusVHost()
 	c := &Caches{
 		translatedIngresses: make(map[string]*translatedIngress),
 		clusters:            newClustersCache(logger.Named("cluster-cache")),
 		clustersToIngress:   make(map[string][]string),
 		logger:              logger,
+		statusVirtualHost:   &statusVirtualHost,
```

Comment:
Given that it's not used anywhere else, we could do `&statusVHost()` here and delete the assignment above.

Reply:
Nope, that notation doesn't work. If we merge #309 first though, we can do that.